pkgs/top-level/python-packages.nix

```
@@ -6383,8 +6383,6 @@ in {
 
   maildir-deduplicate = callPackage ../development/python-modules/maildir-deduplicate { };
 
```

deprecation story
```suggestion

  mps-youtube = throw "mps-youtube has moved to all-packages, and no longer available as a python module."
```

@FRidh does this look right?

---

Please add a date, like is done in `aliases.nix`.